### DISCUSSION TITLE
Debounce the onChange callback

### DISCUSSION THREAD

**Discussion by Teebo** [codex-team/editor.js#1903] [Q&A]
How can I debounce the onChange callback.
I would like to autosave the content from the editor, but I would like to wait a few seconds before the autosave.

**Comment by Teebo** [comment]
Re-opened issue in case someone wants to share something.

**Comment by Teebo** [comment]
@stakable the solution you have suggested does not fit my use case, do you know how I can contribute to the project?
I saw where I can fix this from the package itself.

**Comment by Teebo** [comment]
@stakable I have implemented the solution, see pull request https://github.com/codex-team/editor.js/pull/1222

**Comment by marek-miotelka** [comment]
Hi, what if I want to lower the debounce time?

I want to implement character limit feature where user shouldn't be able to add text over the specified limit.

WIth current debounce time he is able to do it, and then the content he added is removed.
Also the character counter is lagging, cause it's picking up the changes 450ms after a user stopped typing.

**Reply by neSpecc** [reply]
@marek-miotelka I recently tried to do that, and there were no problems: https://codex.so/editorjs-max-length
